metricbeat/mb/testing/data_generator.go

```diff
@@ -78,7 +78,6 @@ func StandardizeEvent(ms mb.MetricSet, e mb.Event, modifiers ...mb.EventModifier
 	}
 
 	e.Timestamp = startTime
-	e.Namespace = ms.Registration().Namespace
```

Comment:
@andrewkroh Can you check if this change has unintended side effects on the auditbeat side?

Reply:
This will affect MetricSets that provide a Namespace at registration time. I think that by adding a conditional this should be safe.

```
if e.Namespace == "" {
    e.Namespace = ms.Registration().Namespace
}
```

Reply:
@andrewkroh Add the if clause and switched node_stats to also use the registry for the namespace as it's constant. I will do the same for node_stats on the ES side.